*** tests2/A-06-05.out	2017-12-07 08:13:15.185295200 +0000
--- tests2/A-06-05.outhyp	2017-12-07 11:13:24.126261800 +0000
***************
*** 11,28 ****
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
! Itinerário 1 para 2017-12-25 @ 12.20
! Serviço #696 @ 12.20
! 19:06 Évora
! 19:17 Casa Branca
! 19:31 Vendas Novas
! 19:53 Pinhal Novo
! 20:14 Pragal
! 20:24 Lisboa - Sete Rios
! 20:28 Lisboa - Entrecampos
! 20:36 Lisboa - Oriente
! Número do itinerário: Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
--- 11,17 ----
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
