*** tests2/A-06-06.out	2017-12-07 08:13:15.193316100 +0000
--- tests2/A-06-06.outhyp	2017-12-07 14:19:08.895041500 +0000
***************
*** 11,44 ****
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
! Itinerário 1 para 2017-12-15 @ 8.90
! Serviço #6403 @ 8.90
! 05:41 Lisboa - Santa Apolónia
! 05:55 Lisboa - Entrecampos
! 05:58 Lisboa - Sete Rios
! 06:13 Agualva-Cacém
! 06:18 Mira Sintra-Meleças
! 06:22 Telhal
! 06:26 Sabugo
! 06:31 Pedra Furada
! 06:35 Mafra
! 06:42 Malveira
! 06:47 Jerumelo
! 06:52 Sapataria
! 06:55 Pero Negro
! 06:58 Zibreira
! 07:01 Feliteira
! 07:05 Dois Portos
! 07:10 Runa
! 07:15 Torres Vedras
! 07:23 Ramalhal
! 07:30 Outeiro
! 07:41 Bombarral
! 07:46 Paúl
! 07:50 São Mamede
! 07:54 Dagorda-Peniche
! Número do itinerário: Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
--- 11,17 ----
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
