*** tests2/A-08-04.out	2017-12-07 08:13:15.272193700 +0000
--- tests2/A-08-04.outhyp	2017-12-09 18:43:49.052211800 +0000
***************
*** 13,25 ****
  0 - Sair
  Escolha uma opção: Identificador do passageiro: == Passageiro 0: Obi-Wan ==
  
! Itinerário 1 para 2017-09-19 @ 51.50
  Serviço #180 @ 51.50
  05:47 Porto - Campanhã
  08:10 Casa Branca
  11:23 Faro
  
! Itinerário 2 para 2017-10-14 @ 32.76
  Serviço #180 @ 21.92
  05:47 Porto - Campanhã
  08:10 Casa Branca
--- 13,38 ----
  0 - Sair
  Escolha uma opção: Identificador do passageiro: == Passageiro 0: Obi-Wan ==
  
! 
! Itinerário 1 para 2017-10-17 @ 51.50
! Serviço #180 @ 51.50
! 05:47 Porto - Campanhã
! 08:10 Casa Branca
! 11:23 Faro
! 
! Itinerário 2 para 2017-09-19 @ 51.50
! Serviço #180 @ 51.50
! 05:47 Porto - Campanhã
! 08:10 Casa Branca
! 11:23 Faro
! 
! Itinerário 3 para 2017-10-18 @ 51.50
  Serviço #180 @ 51.50
  05:47 Porto - Campanhã
  08:10 Casa Branca
  11:23 Faro
  
! Itinerário 4 para 2017-10-19 @ 32.76
  Serviço #180 @ 21.92
  05:47 Porto - Campanhã
  08:10 Casa Branca
***************
*** 27,33 ****
  17:08 Casa Branca
  18:36 Lisboa - Oriente
  
! Itinerário 3 para 2017-10-15 @ 32.76
  Serviço #180 @ 21.92
  05:47 Porto - Campanhã
  08:10 Casa Branca
--- 40,46 ----
  17:08 Casa Branca
  18:36 Lisboa - Oriente
  
! Itinerário 5 para 2017-10-15 @ 32.76
  Serviço #180 @ 21.92
  05:47 Porto - Campanhã
  08:10 Casa Branca
***************
*** 35,53 ****
  17:08 Casa Branca
  18:36 Lisboa - Oriente
  
! Itinerário 4 para 2017-10-17 @ 51.50
! Serviço #180 @ 51.50
! 05:47 Porto - Campanhã
! 08:10 Casa Branca
! 11:23 Faro
! 
! Itinerário 5 para 2017-10-18 @ 51.50
! Serviço #180 @ 51.50
! 05:47 Porto - Campanhã
! 08:10 Casa Branca
! 11:23 Faro
! 
! Itinerário 6 para 2017-10-19 @ 32.76
  Serviço #180 @ 21.92
  05:47 Porto - Campanhã
  08:10 Casa Branca
--- 48,54 ----
  17:08 Casa Branca
  18:36 Lisboa - Oriente
  
! Itinerário 6 para 2017-10-14 @ 32.76
  Serviço #180 @ 21.92
  05:47 Porto - Campanhã
  08:10 Casa Branca
