*** tests2/A-08-03.out	2017-12-07 08:13:15.272193700 +0000
--- tests2/A-08-03.outhyp	2017-12-07 11:13:27.422352600 +0000
***************
*** 11,25 ****
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: == Passageiro 0: Obi-Wan ==
! 
! Itinerário 1 para 2017-10-18 @ 32.76
! Serviço #180 @ 21.92
! 05:47 Porto - Campanhã
! 08:10 Casa Branca
! Serviço #694 @ 10.84
! 17:08 Casa Branca
! 18:36 Lisboa - Oriente
  Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
--- 11,17 ----
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: O passageiro 0 não tem itinerários.
  Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
