*** tests2/A-08-01.out	2017-12-07 08:13:15.256569000 +0000
--- tests2/A-08-01.outhyp	2017-12-07 11:07:15.593784800 +0000
***************
*** 20,42 ****
  5 - Gestão de passageiros
  6 - Gestão de itinerários
  0 - Sair
- Escolha uma opção: Menu de gestão de itinerários
- 1 - Mostrar todos os itinerários
- 2 - Mostrar itinerários associados a um passageiro
- 3 - Registar itinerário para um passageiro
- 0 - Sair
- Escolha uma opção: Identificador do passageiro: O passageiro 1 não tem itinerários.
- Menu de gestão de itinerários
- 1 - Mostrar todos os itinerários
- 2 - Mostrar itinerários associados a um passageiro
- 3 - Registar itinerário para um passageiro
- 0 - Sair
- Escolha uma opção: Menu principal
- 1 - Reiniciar
- 2 - Abrir
- 3 - Guardar
- 4 - Consulta de serviços
- 5 - Gestão de passageiros
- 6 - Gestão de itinerários
- 0 - Sair
  Escolha uma opção: 
\ No newline at end of file
--- 20,23 ----
