*** tests2/A-09-02.out	2017-12-07 08:13:15.301160000 +0000
--- tests2/A-09-02.outhyp	2017-12-07 14:42:47.313417600 +0000
***************
*** 11,30 ****
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
! Itinerário 1 para 2017-12-25 @ 16.04
! Serviço #590 @ 9.64
! 07:02 Lisboa - Oriente
! 07:10 Lisboa - Entrecampos
! 07:57 Poceirão
! 08:06 Pegoes
! 08:10 São João das Craveiras
! 08:21 Vendas Novas
! Serviço #690 @ 6.40
! 08:31 Vendas Novas
! 08:53 Pinhal Novo
! 09:14 Pragal
! Número do itinerário: Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
--- 11,72 ----
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): Menu de gestão de itinerários
! 1 - Mostrar todos os itinerários
! 2 - Mostrar itinerários associados a um passageiro
! 3 - Registar itinerário para um passageiro
! 0 - Sair
! Escolha uma opção: 0|Azura|NORMAL|0|0.00|00:00
! == Passageiro 0: Azura ==
! 
! 1|Boethiah|NORMAL|0|0.00|00:00
! == Passageiro 1: Boethiah ==
! 
! 2|Clavicus Vile|NORMAL|0|0.00|00:00
! == Passageiro 2: Clavicus Vile ==
! 
! 3|Hermaeus Mora|NORMAL|0|0.00|00:00
! == Passageiro 3: Hermaeus Mora ==
! 
! 4|Hircine|NORMAL|0|0.00|00:00
! == Passageiro 4: Hircine ==
! 
! 5|Malacath|NORMAL|0|0.00|00:00
! == Passageiro 5: Malacath ==
! 
! 6|Mehrunes Dagon|NORMAL|0|0.00|00:00
! == Passageiro 6: Mehrunes Dagon ==
! 
! 7|Mephala|NORMAL|0|0.00|00:00
! == Passageiro 7: Mephala ==
! 
! 8|Meridia|NORMAL|0|0.00|00:00
! == Passageiro 8: Meridia ==
! 
! 9|Molag Bal|NORMAL|0|0.00|00:00
! == Passageiro 9: Molag Bal ==
! 
! 10|Namira|NORMAL|0|0.00|00:00
! == Passageiro 10: Namira ==
! 
! 11|Nocturnal|NORMAL|0|0.00|00:00
! == Passageiro 11: Nocturnal ==
! 
! 12|Peryite|NORMAL|0|0.00|00:00
! == Passageiro 12: Peryite ==
! 
! 13|Sanguine|NORMAL|0|0.00|00:00
! == Passageiro 13: Sanguine ==
! 
! 14|Sheogorath|NORMAL|0|0.00|00:00
! == Passageiro 14: Sheogorath ==
! 
! 15|Vaermina|NORMAL|0|0.00|00:00
! == Passageiro 15: Vaermina ==
! 
! 16|Jyggalag|NORMAL|0|0.00|00:00
! == Passageiro 16: Jyggalag ==
! Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
***************
*** 43,49 ****
  3 - Registar passageiro
  4 - Mudar nome de passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: 1|Boethiah|NORMAL|1|16.04|02:12
  Menu de gestão de passageiros
  1 - Mostrar todos os passageiros
  2 - Mostrar passageiro
--- 85,91 ----
  3 - Registar passageiro
  4 - Mudar nome de passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: 1|Boethiah|NORMAL|0|0.00|00:00
  Menu de gestão de passageiros
  1 - Mostrar todos os passageiros
  2 - Mostrar passageiro
