*** tests2/A-09-04.out	2017-12-07 08:13:15.314187600 +0000
--- tests2/A-09-04.outhyp	2017-12-07 11:13:28.598794200 +0000
***************
*** 12,18 ****
  3 - Registar passageiro
  4 - Mudar nome de passageiro
  0 - Sair
! Escolha uma opção: 0|Obi-Wan|NORMAL|4|149.79|44:03
  1|Yoda|NORMAL|0|0.00|00:00
  Menu de gestão de passageiros
  1 - Mostrar todos os passageiros
--- 12,18 ----
  3 - Registar passageiro
  4 - Mudar nome de passageiro
  0 - Sair
! Escolha uma opção: 0|Obi-Wan|NORMAL|0|0.00|00:00
  1|Yoda|NORMAL|0|0.00|00:00
  Menu de gestão de passageiros
  1 - Mostrar todos os passageiros
***************
*** 33,44 ****
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
! Itinerário 1 para 2018-01-01 @ 2400.00
! Serviço #5500 @ 2400.00
! 15:48 Elvas
! 18:25 Entroncamento
! Número do itinerário: Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
--- 33,46 ----
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): Menu de gestão de itinerários
! 1 - Mostrar todos os itinerários
! 2 - Mostrar itinerários associados a um passageiro
! 3 - Registar itinerário para um passageiro
! 0 - Sair
! Escolha uma opção: 0|Obi-Wan|NORMAL|0|0.00|00:00
! 1|Yoda|NORMAL|0|0.00|00:00
! Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
***************
*** 57,63 ****
  3 - Registar passageiro
  4 - Mudar nome de passageiro
  0 - Sair
! Escolha uma opção: 0|Obi-Wan|ESPECIAL|5|2549.79|46:40
  1|Yoda|NORMAL|0|0.00|00:00
  Menu de gestão de passageiros
  1 - Mostrar todos os passageiros
--- 59,65 ----
  3 - Registar passageiro
  4 - Mudar nome de passageiro
  0 - Sair
! Escolha uma opção: 0|Obi-Wan|NORMAL|0|0.00|00:00
  1|Yoda|NORMAL|0|0.00|00:00
  Menu de gestão de passageiros
  1 - Mostrar todos os passageiros
***************
*** 78,90 ****
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
! Itinerário 1 para 2019-01-01 @ 51.50
! Serviço #180 @ 51.50
! 05:47 Porto - Campanhã
! 08:10 Casa Branca
! 11:23 Faro
! Número do itinerário: Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
--- 80,93 ----
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
  0 - Sair
! Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): Menu de gestão de itinerários
! 1 - Mostrar todos os itinerários
! 2 - Mostrar itinerários associados a um passageiro
! 3 - Registar itinerário para um passageiro
! 0 - Sair
! Escolha uma opção: 0|Obi-Wan|NORMAL|0|0.00|00:00
! 1|Yoda|NORMAL|0|0.00|00:00
! Menu de gestão de itinerários
  1 - Mostrar todos os itinerários
  2 - Mostrar itinerários associados a um passageiro
  3 - Registar itinerário para um passageiro
***************
*** 103,109 ****
  3 - Registar passageiro
  4 - Mudar nome de passageiro
  0 - Sair
! Escolha uma opção: 0|Obi-Wan|ESPECIAL|6|2575.54|52:16
  1|Yoda|NORMAL|0|0.00|00:00
  Menu de gestão de passageiros
  1 - Mostrar todos os passageiros
--- 106,112 ----
  3 - Registar passageiro
  4 - Mudar nome de passageiro
  0 - Sair
! Escolha uma opção: 0|Obi-Wan|NORMAL|0|0.00|00:00
  1|Yoda|NORMAL|0|0.00|00:00
  Menu de gestão de passageiros
  1 - Mostrar todos os passageiros
