*** tests/A-01-01.out	2017-11-06 22:18:20.000000000 +0000
--- tests/A-01-01.outhyp	2017-11-18 18:38:05.049576500 +0000
***************
*** 12,18 ****
  3 - Mostrar serviços com início em estação
  4 - Mostrar serviços com término em estação
  0 - Sair
! Escolha uma opção: Serviço #694 @ 12.20
  16:57 Évora
  17:08 Casa Branca
  17:22 Vendas Novas
--- 12,18 ----
  3 - Mostrar serviços com início em estação
  4 - Mostrar serviços com término em estação
  0 - Sair
! Escolha uma opção: Serviço #694 @ 12.2
  16:57 Évora
  17:08 Casa Branca
  17:22 Vendas Novas
***************
*** 25,31 ****
  18:24 Lisboa - Sete Rios
  18:28 Lisboa - Entrecampos
  18:36 Lisboa - Oriente
! Serviço #696 @ 12.20
  19:06 Évora
  19:17 Casa Branca
  19:31 Vendas Novas
--- 25,31 ----
  18:24 Lisboa - Sete Rios
  18:28 Lisboa - Entrecampos
  18:36 Lisboa - Oriente
! Serviço #696 @ 12.2
  19:06 Évora
  19:17 Casa Branca
  19:31 Vendas Novas
***************
*** 34,40 ****
  20:24 Lisboa - Sete Rios
  20:28 Lisboa - Entrecampos
  20:36 Lisboa - Oriente
! Serviço #6403 @ 9.50
  05:41 Lisboa - Santa Apolónia
  05:55 Lisboa - Entrecampos
  05:58 Lisboa - Sete Rios
--- 34,40 ----
  20:24 Lisboa - Sete Rios
  20:28 Lisboa - Entrecampos
  20:36 Lisboa - Oriente
! Serviço #6403 @ 9.5
  05:41 Lisboa - Santa Apolónia
  05:55 Lisboa - Entrecampos
  05:58 Lisboa - Sete Rios
